### ISSUE TITLE
The All Results page describes missing student IDs as "null" 

### ISSUE THREAD

**Issue by jsorva** [apluslms/a-plus#1272]
My Gmail test account now shows up on the "All Results" page as shown below — as do various student accounts. "null" isn’t the best user-facing string. (This is not very important, since it’s only teachers that see this, I presume.)

![image](https://github.com/apluslms/a-plus/assets/8523129/f756899a-9596-45bf-8c3f-e95482934c26)

**Comment by Mankro** [comment]
MOOC students don't have student ids, thus the value is missing. The "All results" table uses the word null in this case, but sure, the word could be changed.

It has been like this for a long time. This is not a new change.

**Comment by ashi006** [comment]
@jsorva would the hyphen "-" be a good replacement for null or should we leave it empty?

**Comment by jsorva** [comment]
> @jsorva would the hyphen "-" be a good replacement for null or should we leave it empty?

Hi! I don’t have strong opinions about this. I’d probably put either a dash, like you suggest, or a string like "none" or "n/a". If you go with a dash, an em dash (—) would be more appropriate than a hyphen.
